pkg/webhookreceiver/handlers_test.go

```diff
@@ -0,0 +1,82 @@
+package webhookreceiver
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"testing"
+
+	"github.com/gorilla/mux"
+	. "github.com/onsi/ginkgo"
```

Comment:
We've just made the switch to Ginkgov2 in OSDE2E in the last week.. since this would be the introduction of Ginkgo into the OA codebase, might be good to start off with v2 from the get-go.  

On the whole this should be pretty easy and not need many (hopefully no) changes.

https://onsi.github.io/ginkgo/MIGRATING_TO_V2 for the details.

Reply:
Added ginkgo v2